docs/src/solutions.md

```diff
@@ -353,3 +353,25 @@ for i in 2:result_count(model)
     end
 end
 ```
+
+## Checking feasibility of solutions
+
+To check the feasibility of a primal solution, use
+[`primal_feasibility_report`](@ref). The returned `report` is `nothing` if the
+point is feasible. Otherwise, it is a dictionary mapping the infeasible
```

Comment:
What do "feasible" and "infeasible" mean? I don't think you can ignore discussion of tolerances here.

Reply:
I had a go at revising.